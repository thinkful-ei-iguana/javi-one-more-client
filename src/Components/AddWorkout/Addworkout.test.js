import React from 'react';
import ReactDOM from 'react-dom';
import { BrowserRouter } from 'react-router-dom'
import Addworkout from '../Addworkout/Addworkout';
import renderer from 'react-test-renderer'


describe('add workout' , () => {


  it('renders without crashing', () => {
    const div = document.createElement('div');
    ReactDOM.render(<BrowserRouter><Addworkout /></BrowserRouter>, div);
    ReactDOM.unmountComponentAtNode(div);
  });

  it('renders the UI as expected', () => {
    const tree = renderer
      .create(<BrowserRouter><Addworkout /></BrowserRouter>)
      .toJSON()
    expect(tree).toMatchSnapshot();
  })

  
})