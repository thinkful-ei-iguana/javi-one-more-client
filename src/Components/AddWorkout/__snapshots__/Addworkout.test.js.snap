// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`add workout renders the UI as expected 1`] = `
<div
  className="addworkout-container"
>
  <div
    className="header-container"
  >
    <header
      className="App-header"
    >
      <a
        href="/"
        onClick={[Function]}
      >
        <span>
          One More
        </span>
      </a>
      <a
        href="/workoutPage"
        onClick={[Function]}
      >
        <span
          className="enter-app"
        >
          Enter App
        </span>
      </a>
    </header>
  </div>
  <form
    onSubmit={[Function]}
  >
    <fieldset>
      <legend>
        Workout
      </legend>
      <div>
        <div
          className="form-input"
        >
          <label
            htmlFor="title"
          >
            Title:
          </label>
          <input
            id="title"
            name="title"
            placeholder="back"
            required={true}
            type="text"
          />
        </div>
        <div
          className="form-input"
        >
          <label
            htmlFor="workout1"
          >
            workout:
          </label>
          <input
            id="workout1"
            name="workout1"
            placeholder="back extensions"
            required={true}
            type="text"
          />
        </div>
        <div
          className="form-input"
        >
          <label
            htmlFor="lbs"
          >
            lbs:
          </label>
          <input
            id="lbs"
            name="lbs"
            placeholder="50lbs"
            type="number"
          />
        </div>
        <div
          className="form-input"
        >
          <label
            htmlFor="set1"
          >
            set1:
          </label>
          <input
            id="set1"
            name="set1"
            placeholder="10reps"
            type="number"
          />
        </div>
        <div
          className="form-input"
        >
          <label
            htmlFor="set2"
          >
            set2:
          </label>
          <input
            id="set2"
            name="set2"
            placeholder="10reps"
            type="number"
          />
        </div>
        <div
          className="form-input"
        >
          <label
            htmlFor="set3"
          >
            set3:
          </label>
          <input
            id="set3"
            name="set3"
            placeholder="10reps"
            type="number"
          />
        </div>
      </div>
      <div
        className="button-container"
      >
        <button
          onClick={[Function]}
          type="button"
        >
          Cancel
        </button>
        <button
          type="submit"
        >
          Save
        </button>
      </div>
    </fieldset>
  </form>
  <div>
    <footer>
      copyright 2019, one more
    </footer>
  </div>
</div>
`;
